draft-ietf-scitt-architecture.md

```diff
@@ -274,7 +275,7 @@ A Transparent Statement remains a valid Signed Statement, and may be registered
 Verifier:
 
 : organizations, stakeholders, and users involved in validating supply chain Artifacts.
-Verifiers consume Transparent Statements, verifying their proofs and inspecting the Statement payload, either before using corresponding Artifacts, or later to audit an Artifact's provenance on the supply chain.
+Verifiers analyze Transparent Statements, verifying their proofs and inspecting the Statement payload, either before using corresponding Artifacts, or later to audit an Artifact's provenance on the supply chain.
```

Comment:
nice clarification... 👍

Reply:
I thought `Verifiers verify and validate Transparent Statements, verifying ..." would be unpleasantly redundant but the other usage of "consume" didn't describe what they are doing with it, so I hope others enjoy that tweak as well.